src/mount/fuse/mount_config.cc

```diff
@@ -303,30 +303,48 @@ void usage(const char *progname) {
 	printf("\nFUSE options:\n");
 	fuse_lowlevel_help();
 	printf(
-	"    -o max_write=N\n"
-	"    -o max_readahead=N\n"
-	"    -o max_background=N\n"
-	"    -o congestion_threshold=N\n"
-	"    -o sync_read \n"
-	"    -o async_read \n"
-	"    -o atomic_o_trunc \n"
-	"    -o no_remote_lock \n"
-	"    -o no_remote_flock \n"
-	"    -o no_remote_posix_lock \n"
-	"    -o splice_write \n"
-	"    -o no_splice_write \n"
-	"    -o splice_move \n"
-	"    -o no_splice_move \n"
-	"    -o splice_read \n"
-	"    -o no_splice_read \n"
-	"    -o auto_inval_data \n"
-	"    -o no_auto_inval_data \n"
-	"    -o readdirplus=(yes,no,auto)\n"
-	"    -o async_dio \n"
-	"    -o no_async_dio \n"
-	"    -o writeback_cache \n"
-	"    -o no_writeback_cache \n"
-	"    -o time_gran=N\n"
+"    -o rw                  mount the filesystem read-write\n"
+"    -o ro                  mount the filesystem read-only.\n"
+"    -o suid                allow set-user-identifier or set-group-identifier "
+				"bits to take effect\n"
+"    -o nosuid              do not allow set-user-identifier or set-group-identifier " 
+				"bits to take effect\n"
+"    -o dev                 interpret character or block special devices on the "
+				"filesystem\n"
+"    -o nodev               do not interpret character or block special devices on "
+				"the file system\n"
+"    -o exec                permit execution of binaries\n"
+"    -o noexec              do  not  permit direct execution of any binaries on the " 
+				"mounted filesystem\n"
+"    -o max_write=N         set the maximum number of bytes  in  a  single  write  "
+				"operation. The  default  is 128kbytes. Note, that due to various "
```

Comment:
Use `KB` or `kB` to refer to kilobytes. Please, extend to the rest of the documentation.

Usage example:
`The default is 128KB.`

Reply:
Done